src/book.cpp

```diff
@@ -41,11 +41,17 @@ Book::Book() :
   m_readOnly(false)
 {
 }
+
 /* Destructor */
 Book::~Book()
 {
 }
 
+Book::Illustrations Book::getIllustrations() const
+{
+  return Illustrations(m_illustrations.begin(), m_illustrations.end());
```

Comment:
We don't need to explicitly create a `Illustrations` now. Copy constructor should be enough.

Reply:
See https://github.com/kiwix/libkiwix/pull/630#discussion_r744215633